.cirrus.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+test_task:
+  alias: "build_and_test"
```

Comment:
The alias name needs to match the map name, so this should be changed to `alias: test` or the map should be renamed to `build_and_test_task`.  Getting this wrong will break `hack/get_ci_vm.sh` assuming it's needed in the future.